packages/twenty-server/src/engine/api/rest/controllers/core-batch.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Post, Req, Res } from '@nestjs/common';
+
+import { Request, Response } from 'express';
+
+import { CoreService } from 'src/engine/api/rest/services/core.service';
+import { cleanGraphQLResponse } from 'src/engine/api/rest/utils/clean-graphql-response.utils';
+
+@Controller('rest/batch/*')
+export class CoreBatchController {
```

Comment:
actually it should be named coreService. I'm not sure CoreService is a good name overall: RestApiCoreService?

Reply:
The point is everything should be prefixed by RestApi... I just simplified by removing this repetition

Reply:
But I will think about a less generic name

Reply:
I think its good to prefix it. It will improve developer experience to have uniquely named services. It's fine to have a bit longer name